python/lsst/ts/MTMount/mock/axis_device.py

```diff
@@ -75,6 +76,9 @@ def __init__(self, controller, device_id, start_position=0):
             start_position=start_position,
         )
         self._monitor_move_task = asyncio.Future()
+        self._monitor_move_task.set_result(None)
```

Comment:
This creates task and mark it as done. Then as far as I understand it's testing if the task isn't done to do some actions. I prefer to have the task set in constructor as None, marking it was never launched. Then use if <task> is None or <task>.done() to check whatever the task is completed.

Reply:
I once used that technique long ago but came to *strongly* prefer to always have the instance be a Future or Task (they share the part of the API that matters). That saves a host of "if task is None" guard tests in the code. In fact I've pushed hard for everyone else on the project to use this technique and even offer `salobj.make_done_task()` for this purpose. I use that when the code already imports salobj for some other reason (which is most of the time -- i.e. this is a common pattern in CSCs).